platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -416,6 +422,18 @@ public void updateForLocation(
     }
   }
 
+  private Object sanitizeUIConfiguration(
+      String pid, String configEntryKey, Object configEntryValue) {
+    if (UI_CONFIG_PID.equals(pid)
+        && ("color".equals(configEntryKey) || "background".equals(configEntryKey))
+        && (Arrays.stream(ILLEGAL_CHARACTERS)
+            .parallel()
+            .anyMatch(String.valueOf(configEntryValue)::contains))) {
```

Comment:
❓  I think it's faster to use a `Set<String>` instead for the `ILLEGAL_CHARACTERS` collection, and parallel streaming the config entry value itself, leveraging the `contains` method of `Set` for more power as opposed to the `contains` method of `String`.